AvalancheGo 1.9.3 (#1067)
ava-labs/avalanche-docs: docs/nodes/maintain/subnet-configs.md

This is a node-specific configuration. Every validator of this Subnet has to use this configuration in order to create a full private Subnet.
:::

#### `proposerMinBlockDelay` (time.Duration):

> The minimum delay enforce in building snowman++ blocks. Default is set to 1 second.

Reviewer: ```suggestion
The minimum delay performed when building snowman++ blocks. Default is set to 1 second.
```

> As one of the ways to control network congestion, Snowman++ makes sure that no two blocks can be build before a `proposerMinBlockDelay` seconds delay. Some high-performance custom VM may find this too strict. This flag allows tuning of maximal frequency at which blocks can be built.

Reviewer: ```suggestion
As one of the ways to control network congestion, Snowman++ will only build a block `proposerMinBlockDelay` after the parent block's timestamp. Some high-performance custom VM may find this too strict. This flag allows tuning the frequency at which blocks are built.
```
